tests/integration/run_multi_round_benchmark.py

```
@@ -0,0 +1,342 @@
+    agent_config: AgentConfig,
+    benchmark: str,
+    explicit_ids: Optional[Sequence[str]],
+) -> List[str]:
+    if explicit_ids:
+        task_ids: List[str] = []
+        for value in explicit_ids:
+            if not value:
+                continue
+            task_ids.extend([item.strip() for item in str(value).split(",") if item is not None])
+        if not task_ids:
+            raise ValueError("No valid task ids parsed from --task-ids.")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)